providers/class.two-factor-totp.php

```diff
@@ -99,7 +100,7 @@ public function user_two_factor_options( $user ) {
 	 * @param integer $user_id The user ID whose options are being updated.
 	 */
 	public function user_two_factor_options_update( $user_id ) {
-		if ( isset( $_POST['_nonce_user_two_factor_totp_options'] ) ) {
+		if ( ! empty( $_POST['do_totp_authcode'] ) ) {
```

Comment:
This should probably still check for the nonces existence. For example:
`if ( ! empty( $_POST['do_totp_authcode'] ) && isset( $_POST['_nonce_user_two_factor_totp_options'] ) ) {`